src/components/Posts/single-post.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { graphql } from 'react-apollo';
+import { gql } from 'apollo-boost';
+
+class SinglePost extends Component {
```

Comment:
If you are really starting a new project I suggest use [functional based component with hooks](https://www.apollographql.com/docs/react/api/react-hooks/)